doc/manual/overview.xml

```diff
@@ -1562,6 +1562,94 @@ deploy the new configuration.
 
 </section>
 
+<section>
+  <title>Managing keys</title>
+
+  <para>
+    Files in <filename>/nix/store/</filename> are readable by every
+    user on that host, so storing secret keys embedded in nix derivations
+    is insecure. To address this, nixops provides the configuration
+    option <varname>deployment.keys</varname>, which nixops manages
+    separately from the main configuration derivation for each machine.
+  </para>
+
+  <para>
+    Add a key to a machine like so.
+
+    <screen>
+{
+  machine =
+    { config, pkgs, ... }:
+    {
+      deployment.keys.my-secret.text = "shhh this is a secret";
+      deployment.keys.my-secret.user = "myuser";
+      deployment.keys.my-secret.group = "wheel";
+      deployment.keys.my-secret.permissions = "0640";
+    };
+}
+    </screen>
+
+    This will create a file <filename>/run/keys/my-secret</filename>
+    with the specified contents, ownership, and permissions.
+  </para>
+
+  <para>
+    Among the key options, only <varname>text</varname> is required. The
+    <varname>user</varname> and <varname>group</varname> options both default
+    to <literal>"root"</literal>, and <varname>permissions</varname> defaults
+    to <literal>"0600"</literal>.
+  </para>
+
+  <note>
+    <para>
+      Keys in <literal>/run/keys/</literal> are not stored on disk by default.
+    </para>
+  </note>
+
+  <para>
+    Since the <filename>/run</filename> directory is mounted as a tmpfs filesystem,
+    keys stored there from <varname>deployment.keys</varname> are stored in RAM
```

Comment:
I prefer:

> Keys from <varname>deployment.keys</varname> are stored user <filename>/run/</filename> on a temporary filesystem and will not be persisted across a reboot. To send a rebooted machine it's keys, use <command>nixops send-keys</command>. Note that all  <command>nixops</command> commands implicitly upload keys when appropriate,  so manually sending keys should only be necessary after an unattended reboot.

It is a slightly rearranged version that
- Removes a couple implementation details that I didn't think were relevant here.
- Made `nixops send-keys` sentence a little more clear and consistent with terminology.

What do you think?